src/Assets/Icons/Icons.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+export const HeartIcon = () => {
```

Comment:
@RudeSoul @ace4port  Are we allowing the implementation of multiple components in one file? I think it should be one component per file, make a directory for all relative files and then export them using an index.js file.
Doing so will help in the following ways:
- Keeps the code modular which can be used in other projects if needed
- Searching for a specific component in the module will be easier
- Keeps the code clean and easier to review each component

An example of this can be seen in [Material UI](https://github.com/mui-org/material-ui/tree/next/packages/material-ui-icons/lib)